app/Http/Controllers/QuoteProposal/GetAllQuoteProposalsController.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Http\Controllers\QuoteProposal;
+
+use App\Http\Controllers\Controller;
+use App\Models\QuoteProposal;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+
+class GetAllQuoteProposalsController extends Controller
+{
+    public function __invoke(Request $request): JsonResponse
+    {
+        return response()->json(QuoteProposal::all());
+    }
+}
```

Comment:
The endpoint should be `Get all the pending for approval proposes` so the admin can work based on those proposes.